### ISSUE TITLE
Smiling Face Recognition

### ISSUE THREAD

**Issue by abhisheks008** [abhisheks008/DL-Simplified#152]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : Smiling Face Recognition</br>
:red_circle: **Aim** : Create a DL model which will recognize the smiling faces.</br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/chazzer/smiling-or-not-face-data</br>
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by chhaviy** [comment]
Full name : Chhavi Yadav
GitHub Profile Link : https://github.com/chhaviy
Participant ID: NA
Approach for this Project: I will use YOLOv2 with CNN as all the labels in the datasets are labeled in YOLO format.

**Comment by chhaviy** [comment]
Full name : Chhavi Yadav
GitHub Profile Link : https://github.com/chhaviy
Participant ID: NA
Approach for this Project: I will use Tensorflow, Keras and openCV using CNN to identify human face and to classify it as smiling or not smiling.

**Comment by abhisheks008** [comment]
Issue assigned to you @chhaviy

**Comment by abhisheks008** [comment]
Issue has been un-assigned from you due to inactivity.
@chhaviy

**Comment by ayushete02** [comment]
:white_check_mark: 
- Full name : Ayush Shete
- GitHub Profile Link : https://github.com/ayushete02
- Approach for this Project : I will done this by using python 
- What is your participant role? 
Developer

**Comment by adityagoutam23** [comment]
Hey, @abhisheks008 I would love to work on this issue,
Please find all the details, mentioned for the issue to be assigned.

Full name : Aditya Goutam
GitHub Profile Link : https://github.com/adityagoutam23
Email ID : adityagoutam2318@gmail.com
Approaches for this Project :-
Using Haar Cascade for face detection and also CNN for prediction.
What is your participant role: contributor at SWOC 3.0

**Comment by abhisheks008** [comment]
Issue assigned to you @adityagoutam23.
Go ahead!
